docs/running-aggregate-with-docker.md

```diff
@@ -0,0 +1,57 @@
+# Running Aggregate with Docker
+
+## Requirememts
+
+- Install [Docker](https://www.docker.com)
+
+## Quick start - External PostgreSQL instance
```

Comment:
A little more context here would be useful. Maybe quick instructions on how to connect to a Docker PostgreSQL image?

Reply:
I've added some context and an asciinema playblack with the quick start run.

I would like to avoid having to include a small tutorial on connecting different docker images... 

If a user is already using Docker for PostgreSQL odds are that she already knows about how to link containers or she will prefer taking a look to the docker compose build.